ruby/ql/lib/codeql/ruby/security/performance/RegExpTreeView.qll

```diff
@@ -59,6 +59,19 @@ module RegExpFlags {
   }
 }
 
+/**
+ * Provides regular expression patterns.
+ */
+module RegExpPatterns {
+  /**
+   * Gets a pattern that matches common top-level domain names in lower case.
+   */
+  string commonTLD() {
```

Comment:
I saw you got suggestions about this predicate for the other languages; we should apply them here as well.

Reply:
That's right. That should be taken care of by the other pull request which shares the same commit.